src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineStepApi.java

```diff
@@ -34,7 +34,7 @@ public PipelineStepApi(WorkflowRun run) {
                           flowNodeWrapper
                               .getId()), // TODO no need to parse it BO returns a string even though
                       // the datatype is number on the frontend
-                      flowNodeWrapper.getDisplayName(),
+                      flowNodeWrapper.getDisplayName() + " - " + flowNodeWrapper.getArgumentsAsString(),
```

Comment:
is there always arguments? i.e. should `-` be in there unconditionally, getArgumentsAsString is defaulting to ""